packages/kit/package.json

```
@@ -66,7 +66,7 @@
 		"test:integration": "pnpm -r --workspace-concurrency 1 --filter=\"./test/**\" test",
 		"test:cross-platform:dev": "pnpm -r --workspace-concurrency 1 --filter=\"./test/**\" test:cross-platform:dev",
 		"test:cross-platform:build": "pnpm test:unit && pnpm -r --workspace-concurrency 1 --filter=\"./test/**\" test:cross-platform:build",
-		"test:unit": "uvu src \"(spec\\.js|test[\\\\/]index\\.js)\"",
+		"test:unit": "vitest run",
```

```suggestion
		"test:unit": "vitest run",
		"test:unit:watch": "vitest",
```

or even watch mode by default? vitest disables it automatically on CI

---

I don't know if it's worth it. Much easier to type `npx vitest` than `pnpm test:unit:watch`

---

If you are planning to use Vitest in templates, I recommend using just `vitest` in commands (without `run`). There is [a reason](https://github.com/vitest-dev/vitest/discussions/175#discussioncomment-1861798) why Vitest has watch mode enabled by default, and I think it's worth it (we changed how it detects CI, so it's more stable since the answer was given).

---

thanks! we've actually been offering `vitest` in our templates for quite awhile (it's an option that lets you setup playwright, vitest, both, or neither) and do use just the `vitest` command there

---

I looked at using `vitest` rather than `vitest run` here, but the reason it doesn't work is that it hangs `pnpm`